include/pybind11/operators.h

```diff
@@ -7,9 +7,18 @@
     BSD-style license that can be found in the LICENSE file.
 */
 
-#pragma once
+#include <Python.h>
+#include <stdlib.h>
+
+#include <type_traits>
+#include <utility>
 
-#include "pybind11.h"
+#include "pybind11/attr.h"
+#include "pybind11/detail/common.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl_bind.h"
+
+#pragma once
```

Comment:
What's the point of moving `#pragma once` *after* all the include directives? Seems like a waste of I/O operations.

Reply:
Doesn't really make a very big difference, but I have no idea why it moved it down.